azure_pipeline.yaml

```
@@ -13,7 +13,62 @@ variables:
   action:
 
 stages:
+  - stage: Precheck
+    jobs:
+      - job: Validate
+        pool:
+          vmImage: $(agentPool)
+        steps:
+        - task: AzureKeyVault@1
+          inputs:
+            ConnectedServiceName: azurerm-sandbox
+            keyVaultName: infra-vault-nonprod
+            secretsFilter: 'azure-devops-token'
+            runAsPreJob: false
+        - task: Bash@3
```

needs better documentation of what this is for
```suggestion
        - task: Bash@3
          displayName: Prevent parallel run
```

---

Updated